### DISCUSSION TITLE
How can I restrict import specific packages?

### DISCUSSION THREAD

**Discussion by greenlaw110** [detekt/detekt#7366] [Q&A] [answered]
How can I restrict import specific packages? E.g. I would like to prevent my kt source using `org.joda` or `org.junit` packages, how can I make detekt to help me find out the violations?

**Comment by BraisGabin** [comment, marked as answer]
Forbidden Import is the rule you are looking for.

**Reply by schalkms** [reply]
Documentation here
https://detekt.dev/docs/rules/style#forbiddenimport
